toolkits/global/packages/global-author-list/scss/50-components/author-list.scss

```diff
@@ -14,3 +14,13 @@
 .c-author-list--truncated {
 	@include u-list-truncated;
 }
+
+.js {
+	.c-author-list {
+		display: inline;
+	}
+
+	.c-author-list__hide {
+		@include u-hide;
+	}
+}
```

Comment:
OK, as long as we think this is acceptable in the scenario "has js but does not initialise component" then that's fine 😃

Reply:
I 've checked with some of the implementations like Springerlink and inlining doesn't  break anything. That's why I agreed on changing to class toggling rather than style changing